website/docs/docs/build/incremental-microbatch.md

```diff
@@ -0,0 +1,147 @@
+
+dbt will then evaluate which batches need to be loaded, break them up into a SQL query per batch, and load each one independently.
+
+dbt will automatically filter upstream inputs (`source` or `ref`) that define `event_time`, based on the `lookback` and `batch_size` configs for this model.
+
+During standard incremental runs, dbt will process new batches and any according to the configured `lookback` (with one query per batch)
+    
+<Lightbox src="/img/docs/building-a-dbt-project/microbatch/microbatch_lookback.png" title="Configure a lookback to reprocess additional batches during standard incremental runs"/>
+
+If there’s an upstream model that configures `event_time`, but you *don’t* want the reference to it to be filtered, you can specify `ref('upstream_model').render()` to opt-out of auto-filtering.
+
+dbt will evaluate which batches need to be loaded **by processing the current batch (current_timestamp) + any batches in your configured lookback**, break them up into a SQL query per batch, and load them all independently. 
+
+### Backfills
+
+Whether to fix erroneous source data, or retroactively apply a change in business logic, you may need to reprocess a large amount of historical data.
+
+Backfilling a microbatch model is as simple as selecting it to run or build, and specifying a "start" and "end" for `event_time`. As always, dbt will process the batches between the start and end as independent queries.
+
+```bash
+dbt run --event-time-start "2024-09-01" --event-time-end "2024-09-04"
+```
+
+<Lightbox src="/img/docs/building-a-dbt-project/microbatch/microbatch_backfill.png" title="Configure a lookback to reprocess additional batches during standard incremental runs"/>
+
+### Retry
+
+If one or more of your batches fail, you can use `dbt retry` to reprocess _only_ the failed batches.
+
+![Partial retry](https://github.com/user-attachments/assets/f94c4797-dcc7-4875-9623-639f70c97b8f)
+
+### Timezones
+
+For now, dbt assumes that all values supplied are in UTC:
+
+- `event_time`
+- `begin`
+- `--event-time-start` and `--event-time-end`
+
+While we may consider adding support for custom timezones in the future, we also believe that defining these values in UTC makes everyone's lives easier.
+
+## How does `microbatch` compare to other incremental strategies?
+
+Most incremental models rely on the end user (you) to explicitly tell dbt what "new" means, in the context of each model, by writing a filter in an `is_incremental()` block. You are responsibly for crafting this SQL in a way that queries `this` to check when the most recent record was last loaded, with an optional look-back window for late-arriving records. Other incremental strategies will control _how_ the data is being added into the table — whether append-only `insert`, `delete` + `insert`, `merge`, `insert overwrite`, etc — but they all have this in common.
```

Comment:
"Other incremental strategies will control _how_ the data is being added into the table"

maybe add a callout of how microbatch choose which of these "how"s is best?